click/main.js

```diff
@@ -1,72 +1,78 @@
 const app = document.querySelector('.app')
-const firstBox = document.createElement('div')
-firstBox.classList.add('app__box')
-app.appendChild(firstBox)
-const secondBox = document.createElement('div')
-secondBox.classList.add('app__box')
-app.appendChild(secondBox)
+
+const createTimerBox = () => {
+    const firstBox = document.createElement('div')
+    firstBox.classList.add('app__box')
+    app.appendChild(firstBox)
+    return document.querySelector('.app__box')
+}
+
+const createCircleBox = () => {
+    const secondBox = document.createElement('div')
+    secondBox.classList.add('app__box')
+    app.appendChild(secondBox)
+    return document.querySelectorAll('.app__box')[1]
```

Comment:
aa (as above)

Reply:
why are you doing querySelector if you have reference to this element?

simply return:
```
return secondBox
```